lib/rspecq/worker.rb

```diff
@@ -107,7 +108,12 @@ def try_publish_queue!(queue)
       end.map(&:first) & files_to_run
 
       if slow_files.any?
-        puts "Slow files (threshold=#{file_split_threshold}): #{slow_files}"
+        log_event(
+          "Slow files detected (threshold=#{file_split_threshold}): #{slow_files}",
```

Comment:
Threshold is already present in the [context](https://github.com/skroutz/rspecq/pull/16/files#diff-95ff2aa63181520a846e312be076e609R210), but I think this was added here so that it is displayed in STDOUT. Maybe a more uniform approach is required, e.g. mark some context fields as included in STDOUT message as name-value pairs as well `a=1 b=2 ...` ?

Reply:
Yes. I think we should actually use a proper `Logger` and set the levels there accordingly. And use stdout/stderr according to each case (i.e. errors in stderr, debug/info to stdout). I'm also not very happy with the fact that we couple log output with Sentry events (which forces us to submit the very same messages to both), but we can improve this with a subsequent patch. I'd suggest to track this change in a new issue.

Reply:
By the way, created #17 to track this in a separate issue.